### ISSUE TITLE
how to abort already instantiated request?

### ISSUE THREAD

**Issue by ApurvaB** [lexich/redux-api#56]
Hi I am rquesting api using 

````````
api.actions.allUsers.request({},{
    headers: { 'access_token': Token, contains: 'a', skip: 0, limit: 20 }
  }); 
````````

I want to abort this request when I unmount a component.How can I abort the request?

**Comment by lexich** [comment]
You can't abort request. But if you use redux, it's not necessary. You just only update your store.

**Comment by ApurvaB** [comment]
@lexich I am using redux.I am already making changes to make state set to initialState in reducer.Still the request keeps processing.

**Comment by mrbinky3000** [comment]
+1 for the ability to abort a request.  I think that `api.actions.allusers.reset()` should abort any current requests for allusers in addition to resetting the store.

**Comment by mrbinky3000** [comment]
I've looked into cancelling promises in ES6.  You can't yet.  But you can resolve it to something.  In this case `api.actions.allusers.reset()` could resolve any ongoing requests by setting the store to the resetted state.

**Comment by lexich** [comment]
solved #64
